content/module-reference/utility-modules/shared/filmstrip.md

```diff
@@ -12,9 +12,9 @@ The filmstrip can be used to quickly switch between images in any darktable view
 
 The filmstrip can be switched on and off using the shortcut `Ctrl+F`. The height of the filmstrip panel can be changed by clicking and dragging its top border.
 
-Quickly navigate through the images in the filmstrip by scrolling with the mouse. In the darkroom you can change the photo currently being processed by double clicking on another image in the filmstrip.
+Quickly navigate through the images in the filmstrip by scrolling with the mouse. In the darkroom you can change the photo currently being processed by clicking on another image in the filmstrip.
 
-In the darkroom, the image currently being processed is selected and highlighted. Single click to select any other image from the filmstrip without changing the image being processed.
+In the darkroom, the image currently being processed is selected and highlighted. Use Mouse over to select any other image from the filmstrip without changing the image being processed.
```

Comment:
I think either works. Perhaps we should say "hover over or single-click on an image..."

Reply:
Single click now changes the image currently being processed. So what was single click before is mouse over now (only select for ratings etc but do not set it as image to be processed)

Reply:
hmm there's something fishy about that. I'm not sure that's intentional.

Reply:
I've raised an issue https://github.com/darktable-org/darktable/issues/7000

Reply:
Ok so it's `Alt`+click to select an image without changing image, and then `Ctrl`+click and `Shift`+click to extend the selection. Not sure why we have to make it work that way but it seems it's intentional.

Reply:
Let's correct this for now to talk about the existing behaviour and we'll change it back again if it gets changed back as a result of the issue.